### DISCUSSION TITLE
Access to vrstats on Quest

### DISCUSSION THREAD

**Discussion by paulmelis** [mozilla/hubs#3717] [General Questions & Discussion] [answered]
I'm testing with a pretty large model (>1M triangles) on the Quest. I'm amazed that performance is actually pretty good in Oculus Browser, but would like to check the statistics to be sure it's really all being rendered. Is there a way to enable the VR stats (or use `/vrstats`) in Hubs on Quest?

**Comment by johnshaughnessy** [comment, marked as answer]
You can either add `vrstats` as a query string parameter (e.g. https://hubs.mozilla.com/9WpkoHG/pristine-likable-domain?vrstats) or toggle it via a chat command
![image](https://user-images.githubusercontent.com/4072106/105385955-c31d5b80-5bc8-11eb-833f-f0baba498238.png)
If you are in immersive VR mode, you will need to "back out" so that you can interact with the 2D ui (where the chat box is). 

Once it's enabled, you should see a box like this:
![image](https://user-images.githubusercontent.com/4072106/105386151-e7793800-5bc8-11eb-9e90-696a2deaee4c.png)
